packages/react-native-reanimated/android/src/main/cpp/worklets/android/WorkletsModule.h

```diff
@@ -46,19 +46,19 @@ class WorkletsModule : public jni::HybridClass<WorkletsModule> {
   }
 
  private:
-  friend HybridBase;
-  jni::global_ref<WorkletsModule::javaobject> javaPart_;
-  jsi::Runtime *rnRuntime_;
-  std::shared_ptr<WorkletsModuleProxy> workletsModuleProxy_;
-
   explicit WorkletsModule(
-      jni::alias_ref<WorkletsModule::jhybridobject> jThis,
```

Comment:
Turns out we don't need to keep the reference to Java part of `WorkletsModule` in cpp.

Reply:
Does it still have to be a `HybridClass` or can we convert it back to `JavaClass` or something?